web/src/components/error/NotFound.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <div class="NotFound">
+
+    <div class="hero is-light nest-footprints-hero">
```

Comment:
For future reference, to make a Bulma hero that takes up the full screen height, you can append a `is-fullheight` class to the div with the `hero` class.